pygama/lgdo/table.py

```diff
@@ -171,7 +173,17 @@ def get_dataframe(self, cols=None, copy=False):
         df = pd.DataFrame(copy=copy)
         if cols is None: cols = self.keys()
         for col in cols:
-            if not hasattr(self[col],'nda'):
+            if isinstance(self[col], VectorOfVectors):
+                df[col] = self[col].tolist()
```

Comment:
@wisecg will this work? Will pandas make a dataframe from a list of lists of possibly unequal length?